tools/benchmark_tool/README.md

```diff
@@ -111,7 +111,10 @@ usage: benchmark_app.py [-h [HELP]] [-i PATHS_TO_INPUT [PATHS_TO_INPUT ...]] -m
                         [-shape SHAPE] [-data_shape DATA_SHAPE] [-layout LAYOUT] [-nstreams NUMBER_STREAMS] [-latency_percentile LATENCY_PERCENTILE] [-nthreads NUMBER_THREADS]
                         [-pin {YES,NO,NUMA,HYBRID_AWARE}] [-exec_graph_path EXEC_GRAPH_PATH] [-pc [PERF_COUNTS]] [-pcsort {no_sort,sort,simple_sort}] [-pcseq [PCSEQ]]
                         [-inference_only [INFERENCE_ONLY]] [-report_type {no_counters,average_counters,detailed_counters}] [-report_folder REPORT_FOLDER] [-dump_config DUMP_CONFIG]
-                        [-load_config LOAD_CONFIG] [-infer_precision INFER_PRECISION] [-ip {u8,U8,f16,FP16,f32,FP32}] [-op {u8,U8,f16,FP16,f32,FP32}] [-iop INPUT_OUTPUT_PRECISION]
+                        [-load_config LOAD_CONFIG] [-infer_precision INFER_PRECISION]
+                        [-ip {boolean,BOOL,f16,FP16,f32,FP32,f64,FP64,i4,I4,i8,I8,i16,I16,i32,I32,i64,I64,u1,U1,u4,U4,u8,U8,u16,U16,u32,U32,u64,U64,bf16,BF16}]
```

Comment:
1. Can we remove i4/u4bf16 and other types which has no sense to support as input types
2. Can we avoid duplication in upper and lower cases?

Reply:
1. Removed!
2. Removed uppercases, made the arguments non case sensitive